README.md

```diff
@@ -139,10 +139,14 @@ pants test helloworld/translator/translator_test.py -- -k test_unknown_phrase  #
 
 ## Create a PEX binary
 
+The `package` goal requires specifying a target which can be packaged. In this case, the there is a `pex_binary` target with the name `pex_binary` in the `helloworld/BUILD` file.
+
 ```
-pants package helloworld/main.py
+pants package helloworld:pex_binary
```

Comment:
Does pointing `package` at a file (that has a corresponding pex_binary) not work any more?

Reply:
Correct. I got the following error when I ran the original command:

```
% pants package helloworld/main.py
07:55:48.22 [WARN] No applicable files or targets matched. The `package` goal works with these target types:

  * archive
  * pex_binary
  * python_distribution

However, you only specified file arguments with these target types:

  * python_source

Please specify relevant file and/or target arguments. Run `pants --filter-target-type=archive,pex_binary,python_distribution list ::` to find all applicable targets in your project.
```

Reply:
(I think this was the "secondary owner" semantics deprecated in https://github.com/pantsbuild/pants/pull/18737)

Reply:
Blurg. Makes sense though.